### PR TITLE
chore: [IOAPPPEB-15] Move common bonuses code to common folder

### PR DESCRIPTION
**Pull request by francescopersico**
## Short description
Move common bonuses code to common folder

## List of changes proposed in this pull request
- Just file movements

## How to test
No real changes

### CODE REVIEW

**Comment by hevelius** [comment]
LGTM

**Review by forrest57 — APPROVED** [review]

**Review by hevelius — APPROVED** [review]

File: ts/features/bonus/common/BaseTimeoutScreen.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import I18n from "../../../../i18n";
-import { cancelButtonProps } from "./buttons/ButtonConfigurations";
-import { FooterStackButton } from "./buttons/FooterStackButtons";
-import { bonusVacanzeStyle } from "./Styles";
+import image from "../../../../img/wallet/errors/payment-expired-icon.png";
+import { renderInfoRasterImage } from "../../../components/infoScreen/imageRendering";
+import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
+import I18n from "../../../i18n";
+import { cancelButtonProps } from "../bonusVacanze/components/buttons/ButtonConfigurations";
+import { FooterStackButton } from "../bonusVacanze/components/buttons/FooterStackButtons";
```
Comment (CrisTofani) [inline ts/features/bonus/common/BaseTimeoutScreen.tsx:8]: I guess this two components may be moved in  common folder too, they've been reused inside the app

**Review by CrisTofani — CHANGES_REQUESTED** [review]

File: ts/features/bonus/common/BaseTimeoutScreen.tsx (hunk 1 above)
Comment (francescopersico) [reply ts/features/bonus/common/BaseTimeoutScreen.tsx:8]: Yes it is part of another PR because it is not common only to bonuses but also outside.

Comment (CrisTofani) [reply ts/features/bonus/common/BaseTimeoutScreen.tsx:8]: Ok, so we can merge this and then migrate these components in another PR

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**